packages/perseus/src/widgets/measurer/measurer.tsx

```diff
@@ -17,22 +18,10 @@ const defaultImage = {
     left: 0,
 } as const;
 
-type Props = {
+type Props = PerseusMeasurerWidgetOptions & {
     apiOptions: APIOptions;
-    box: [number, number];
-    image: {
-        url?: string;
-        top?: number;
-        left?: number;
-    };
-    showProtractor: boolean;
     protractorX: number;
     protractorY: number;
```

Comment:
Where do protractorX and protractorY come from, if not the widget options?

I suspect the default props are always used on first render, and then protractorX and protractorY get set by an onChange call.

This probably deserves a code comment.